app/views/shared/check_answers/_client_details.html.erb

```diff
@@ -25,6 +25,30 @@
     <% end %>
   <% end %>
 
+  <% if :changed_last_name.in?(attributes) %>
+    <%= summary_list.with_row(html_attributes: { id: "app-check-your-answers__changed_last_name" }) do |row| %>
+      <%= row.with_key(text: t(".changed_last_name"), classes: "govuk-!-width-one-half") %>
+      <%= row.with_value(text: yes_no(applicant.changed_last_name)) %>
+      <%= row.with_action(
+            text: t("generic.change"),
+            href: providers_legal_aid_application_applicant_details_path(@legal_aid_application, anchor: :changed_last_name),
+            visually_hidden_text: "#{t('.aria_prefix')} #{t('.changed_last_name')}",
```

Comment:
Will this visually hidden text mean it reads as "Change client's Has your client ever changed their last name?" If so, can the `aria_prefix` be removed?

Reply:
👍 Thanks @agoldstone93, have removed